autoload/neomake/makers/ft/yaml.vim

```diff
@@ -0,0 +1,12 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#yaml#EnabledMakers()
+    return ['yamllint']
+endfunction
+
+function! neomake#makers#ft#yaml#yamllint()
+    return {
+        \ 'args': ['-f', 'parsable'],
```

Comment:
Consider passing `-j` here. The default setting of 16 concurrency can slam a system on larger projects.